test_utils/fake_catalog_api.py

```diff
@@ -566,6 +570,174 @@
+            'video': None,
+            'key': 'edX+DemoX',
+            'short_description': None,
+            'marketing_url': None,
+            'level_type': None,
+            'course_runs': []
+        }
+    ]
+}
+
+FAKE_SEARCH_ALL_COURSE_RESULT = {
+    "title": "edX Demonstration Course",
+    "min_effort": None,
+    "marketing_url": "course/course-v1:edX+DemoX+Demo_Course/about",
+    "image_url": "https://business.sandbox.edx.org/asset-v1:edX+DemoX+Demo_Course+type"
+                 "@asset+block@images_course_image.jpg",
+    "pacing_type": "instructor_paced",
+    "short_description": None,
+    "subject_uuids": [],
+    "transcript_languages": [],
+    "full_description": None,
+    "seat_types": [
+        "audit",
+        "verified"
+    ],
+    "mobile_available": False,
+    "end": None,
+    "partner": "edx",
+    "max_effort": None,
+    "start": "2013-02-05T05:00:00",
+    "weeks_to_complete": None,
+    "published": True,
+    "content_type": "courserun",
+    "authoring_organization_uuids": [
+        "12de950c-6fae-49f7-aaa9-778c2fbdae56"
+    ],
+    "enrollment_start": None,
+    "staff_uuids": [],
+    "language": None,
+    "number": "DemoX",
+    "type": "verified",
+    "key": "course-v1:edX+DemoX+Demo_Course",
+    "org": "edX",
+    "level_type": None,
+    "program_types": [],
+    "aggregation_key": "courserun:edX+DemoX",
+    "logo_image_urls": [
+        None
+    ],
+    "enrollment_end": None,
+    "availability": "Upcoming"
+}
+
+FAKE_SEARCH_ALL_SHORT_COURSE_RESULT = {
+    "title": "edX Demonstration Course",
+    "full_description": None,
+    "key": "edX+DemoX",
+    "short_description": None,
+    "aggregation_key": "course:edX+DemoX",
+    "content_type": "course"
+}
+
+FAKE_SEARCH_ALL_PROGRAM_RESULT = {
+    "title": "Program Title 1",
+    "marketing_url": "professional-certificate/marketingslug1",
+    "content_type": "program",
+    "card_image_url": "http://wowslider.com/sliders/demo-10/data/images/dock.jpg",
+    "min_hours_effort_per_week": 5,
+    "authoring_organization_uuids": [
+        "12de950c-6fae-49f7-aaa9-778c2fbdae56"
+    ],
+    "hidden": False,
+    "authoring_organizations": [
+        {
+            "marketing_url": None,
+            "homepage_url": None,
+            "tags": [],
+            "certificate_logo_image_url": None,
+            "name": "",
+            "key": "edX",
+            "description": None,
+            "uuid": "12de950c-6fae-49f7-aaa9-778c2fbdae56",
+            "logo_image_url": None
+        }
+    ],
+    "staff_uuids": [],
+    "published": True,
+    "uuid": FAKE_UUIDS[-2],
+    "max_hours_effort_per_week": 10,
+    "subject_uuids": [],
+    "weeks_to_complete_min": None,
+    "type": "Professional Certificate",
+    "language": [
+        "English"
+    ],
+    "partner": "edx",
+    "subtitle": "Program Subtitle 1",
+    "status": "active",
+    "weeks_to_complete_max": None,
+    "aggregation_key": "program:" + FAKE_UUIDS[-2]
```

Comment:
@itsjeyd No good reason -- because I previously had 10 UUIDs I had to deal with, I decided to think of the ends rather than random numbers in the middle. But now with 5, I'll keep things positive.